horizons/world/units/ship.py

```diff
@@ -154,6 +154,7 @@ def _update_buoy(self, remove_only=False):
 
 		ship_id = self.worldid
 		session = self.session # this has to happen here,
+
```

Comment:
Sorry, no. The comment belongs to the session = line, but the session = line belongs to the function. So imho there shouldn't be any newline.